tests/test_bootstrap_wizard.sh

```
@@ -0,0 +1,311 @@
+set +e
+"$SCRIPT" some-repo --project banana >/dev/null 2>&1
+ec=$?
+set -e
+[ "$ec" -eq 1 ] && pass "invalid --project → exit 1" \
+                || fail "invalid project should exit 1; got $ec"
+
+# ---------------------------------------------------------------------------
+# Test 8: Missing argument for a value-taking flag → exit 1.
+# ---------------------------------------------------------------------------
+set +e
+"$SCRIPT" some-repo --visibility 2>&1 >/dev/null
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)